src/main/steps/c100-rebuild/miam/valid-reason/content.ts

```diff
@@ -0,0 +1,82 @@
+import { YesOrNo } from '../../../../app/case/definition';
+import { TranslationFn } from '../../../../app/controller/GetController';
+import { FormContent } from '../../../../app/form/Form';
+import { isFieldFilledIn } from '../../../../app/form/validation';
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+const en = () => ({
+  lvTitle: 'Do you have valid reasons for not attending a MIAM?',
```

Comment:
@niranjan-301 What's this 'lv' prefix before all these fields ?
Can we please remove it ?